inc/functions-cookies.php

```diff
@@ -0,0 +1,75 @@
+<?php
+// These functions are going to be moved into the plugin
+// upon cross site roll-out
+
+function delete_GA_cookies() {
+	$domain = 'nationalarchives.gov.uk';
+	$cookie_list = ['_ga', '_gid', '_gat_UA-2827241-22', '_gat_UA-2827241-1'];
+    
+	if (isset($_SERVER['HTTP_COOKIE'])) {
+		$cookies = explode(';', $_SERVER['HTTP_COOKIE']);
+		foreach($cookies as $cookie) {
+			$parts = explode('=', $cookie);
+			$name = trim($parts[0]);
+			foreach($cookie_list as $single_cookie) {
+				if($name == $single_cookie) {
+                    unset( $_COOKIE[$name] );
```

Comment:
What would you think about putting the code which clears an individual cookie into a function called `clear_cookie()` which expects a single `$name` argument?

Reply:
Done